transport/api/v1/editor.go

```diff
@@ -48,22 +42,85 @@ func (h *Handler) Register(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	statusCode, err := util.PostSupabaseOTPRequest(payload.Email)
-	if statusCode != 0 && err != nil {
+	requestOTP(w, payload.Email)
+
+	return
+}
+
+func requestOTP(w http.ResponseWriter, email string) {
+	statusCode, err := util.PostSupabaseOTPRequest(email)
+	if statusCode != 200 && err != nil {
 		util.WriteResponse(w, statusCode, "Error sending request to supabase: "+err.Error())
 		return
 	}
 	if err != nil {
-		util.WriteResponse(w, http.StatusBadRequest, "Error sending request to supabase: "+err.Error())
+		util.WriteResponse(w, http.StatusInternalServerError, "Error sending request to supabase: "+err.Error())
 		return
 	}
-	if statusCode != 0 && err == nil {
+	if statusCode != 200 {
 		util.WriteResponse(w, http.StatusInternalServerError, fmt.Sprintf("Something unexpected happened: error is nil but statusCode is %d", statusCode))
 		return
 	}
 	return
 }
 
+func (h *Handler) Verify(w http.ResponseWriter, r *http.Request) {
+	data := r.URL.Query().Get("data")
+	if data == "" {
+		util.WriteResponse(w, http.StatusBadRequest, "No data provided")
+		return
+	}
+
+	params, err := parseData(data)
+	if err != nil {
+		util.WriteResponse(w, http.StatusInternalServerError, err.Error())
+		return
+	}
+
+	if _, err := h.service.CreateEditor(r.Context(), params.Uuid, params.Email); err != nil {
+		util.WriteResponse(w, http.StatusInternalServerError, err.Error())
+		return
+	}
```

Comment:
v připadě úspěchu nevratí ten endpoint to žadný http kod? stejně pro ostatní endpointy